apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProvider.java

```diff
@@ -367,10 +377,13 @@ public Bundle findByPatient(
 
     eobs.sort(R4ExplanationOfBenefitResourceProvider::compareByClaimIdThenClaimType);
 
+    Bundle bundle =
+        TransformerUtilsV2.createBundle(paging, eobs, loadedFilterManager.getTransactionTime());
+
     // Add bene_id to MDC logs
-    TransformerUtilsV2.logBeneIdToMdc(beneficiaryId);
+    LoggingUtils.logBeneIdToMdc(beneficiaryId);
 
-    return TransformerUtilsV2.createBundle(paging, eobs, loadedFilterManager.getTransactionTime());
+    return bundle;
```

Comment:
This also seems like it could be reverted

Reply:
Reverted for all instances